packages/formatter-codeframe/README.md

```diff
@@ -0,0 +1,15 @@
+# Formatter codeframe (`@sonarwhal/formatter-codeframe`)
+
+The `codeframe` formatter shows the results in table format indicating the
+resource, line and column as well as some code where the error was found (if
+applicable):
+
+![Example output for the codeframe formatter](images/codeframe-output.png)
```

Comment:
Move `images/` into `src/` so that they are copied into `dist/` and get publish with the rest.

Reply:
Done, I've changed the `package.json`